app/cdap/text/text-en.yaml

```diff
@@ -3044,22 +3060,15 @@ features:
         hive-db-name-placeholder: "Hive database for this namespace"
         scheduler-queue-name: "Scheduler queue name"
         scheduler-queue-placeholder: "Yarn queue name to be used to submit programs in this namespace"
-        sld-label: "Specify mapping of namespace resources to the existing underlying storage resources."
-        ssd-label: "Namespace mapping"
-      Step3:
+        sld-label: "Specify mapping of namespace resources to underlying Hadoop resources (Only used when deployed on Apache Hadoop)."
+        ssd-label: "Hadoop mapping"
+      Step5:
```

Comment:
Nit: Would it be worthwhile to give the steps stable names for the purposes of these keys? Then we wouldn't need to rename everything if the order changes.

Reply:
I like that idea. Will push a commit with better names shortly